stable_baselines3/dqn/dqn.py

```diff
@@ -0,0 +1,210 @@
+from typing import List, Tuple, Type, Union, Callable, Optional, Dict, Any
+
+import torch as th
+import torch.nn.functional as F
+
+from stable_baselines3.common import logger
+from stable_baselines3.common.base_class import OffPolicyRLModel
+from stable_baselines3.common.type_aliases import GymEnv, MaybeCallback
+from stable_baselines3.common.utils import get_linear_fn
+from stable_baselines3.dqn.policies import DQNPolicy
+
+
+class DQN(OffPolicyRLModel):
+    """
+    Deep Q-Network (DQN)
+
+    Paper: https://arxiv.org/abs/1312.5602
```

Comment:
Please add the nature paper here too

Reply:
I would refer specifically to nature paper, to avoid any confusion with parameters. I think they differ a bit between the two papers.

Reply:
The problem with nature paper is that it is behind a paywall :/

Reply:
Ah, right :/. I see the problem, but I think we should refer to both papers, and say that we use parameters (and default architectures) from the nature version, to avoid any confusion.

Reply:
I was unable to extract the architecture from the paper but I guess it is already implemented in the CnnPolicy? @araffin

Reply:
Yes, [NatureCNN](https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/common/policies.py#L53) is the correct architecture.